test/Microsoft.NET.Publish.Tests/GivenThatWeWantToPublishAHelloWorldProject.cs

```diff
@@ -175,6 +175,80 @@ public static void Main()
                 .And
                 .HaveStdOutContaining("Hello from a netcoreapp2.0.!");
         }
+        
+        //Note: Pre Netcoreapp2.0 standalone activation uses renamed dotnet.exe
+        //      While Post 2.0 we are shifting to using apphost.exe, so both publish needs to be validated
+        [Theory]
+        [InlineData("win-arm")]
+        [InlineData("win8-arm")]
+        [InlineData("win81-arm")]
+        [InlineData("win10-arm")]
```

Comment:
Yes, the list of files will be the same.  Obviously, the contents of those files will be different, but the names will be the same.